.github/workflows/ci.yml

```diff
@@ -46,118 +49,68 @@ jobs:
         with:
           python-version: "3.11"
 
-      - name: Copy Protobuf files to Dockerfile directories
-        run: ./ci/docker/copy-protobuf.sh
+      - name: Install just
+        run: ./ci/scripts/install-just.sh
 
       - name: Build docker containers
-        run: docker compose --profile test build
+        run: just build
 
-      - name: Start docker stack
-        run: DYNAMICTIME=false LOTIME=1000-01-01T00:00:00Z HITIME=9999-12-31T23:59:59Z docker compose up -d --wait --wait-timeout 120
+      - name: Run the unit test
+        run: just unit
 
-      - name: Test loader runs without errors
-        run: DYNAMICTIME=false LOTIME=1000-01-01T00:00:00Z HITIME=9999-12-31T23:59:59Z docker compose run --rm loader
+      - name: Comment coverage
```

Comment:
In other PR, the output of this gives 23 tests, while on this branch it is listed as one. Any idea why?

Reply:
That was after the initial move to a docker container. The coverage was not successfully exported. The file had only one line, on which it failed.

- Initial: https://github.com/EURODEO/e-soh/pull/64#issuecomment-2006973903
- Fixed: https://github.com/EURODEO/e-soh/pull/64#issuecomment-2012304218
- Moved to a separate job because of permissions: https://github.com/EURODEO/e-soh/pull/64#issuecomment-2074201919

In the future, it should be a single comment per PR again. In this PR there are multiple because I changed the job 3 times.